samples/tasks/scenarios/ovn-sandbox/create_sandbox_k12c0ga.json

```diff
@@ -7,10 +7,10 @@
             "name": "OvnSandbox.create_sandbox",
             "args": {
                 "sandbox_create_args": {
-                    "farm": "ovn-farm-node-0",
```

Comment:
json files under 'samples' folder are simply samples, we need not commit environment-specific json files into it.
Seem like 'create_sandbox_k12c0ga.json' is a task file dedicated to your environment, you may keep it in a private repo.